### PR TITLE
adds hulk injector for 6tc to traitor geneticist uplink

### PR DESCRIPTION
**Pull request by boodaliboo**
readds hulk so the code isnt left unused, i cant write descriptions for shit i apologize
:cl:  
rscadd: Added a way to get hulk mutation again
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -2021,6 +2021,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	restricted_roles = list("Assistant")
 	surplus = 0
 
+/datum/uplink_item/role_restricted/theincrediblebulk
+	name = "A Hulk Injector"
+	desc = "A Very Powerful Mutation Discovered in a ancient vault on mars, causes rapid musclegrowth."
```
Comment (redmoogle) [inline code/modules/uplink/uplink_items.dm:2026]:
```suggestion
	name = "Hulk Injector"
	desc = "A very powerful mutation discovered in a ancient vault on mars, When applied the mutation causes rapid muscle growth turning the user in to a hulk."
```

Desc is weird and has grammatical errors
Fix name please

Comment (redmoogle) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
	name = "Hulk Injector"
	desc = "A very powerful mutation discovered in a ancient vault on mars, When applied the mutation causes rapid muscle growth turning the user in to a hulk."
```

name isnt really a suggestion
desc is just fixing of grammatical errors and making it have some fluff to it

**Comment by ghost** [comment]
Antag exclusive mutations is a bad idea since that they can be easily extracted and then mass produced by the crew

**Comment by boodaliboo** [comment]
> Antag exclusive mutations is a bad idea since that they can be easily extracted and then mass produced by the crew
i dont see how that is a bad thing, if the antag traitor decides to extract it and give it to the crew i think thats completely fine, its not like were giving laser eyes out

**Comment by Addust** [comment]
6 TC kinda low to be able to instantly eviscerate walls

**Comment by boodaliboo** [comment]
> 6 TC kinda low to be able to instantly eviscerate walls

you cant click walls i thought, i thought it was random to do so

**Comment by JamieD1** [comment]
Fuck Hulk

**Pull request closed without merging**